python/cudf/cudf/tests/test_concat.py

```diff
@@ -1997,3 +1997,38 @@ def test_concat_dict_incorrect_type_index(d):
         match="cannot concatenate a dictionary containing indices",
     ):
         cudf.concat(d, axis=1)
+
+
+@pytest.mark.parametrize(
+    "axis",
+    [
+        0,
+        pytest.param(
+            1,
+            marks=pytest.mark.xfail(
+                reason=("cannot concatenate indices across axis 1")
+            ),
+        ),
```

Comment:
*issue*: An `xfail` mark should be used where a test is currently failing due to some known bug. In this case, we instead (if anything) want to assert that this usage raises the expected exception.

Unfortunately, this is slightly fiddly because one cannot add a `pytest.raises` mark to the parameter.

The preferred pattern is:

```
from contextlib import nullcontext
...

@pytest.mark.parametrize(
    "axis,exception",
    [
        (0, nullcontext()),
        (1, pytest.raises(...)),
    ]
)
def test_concat_index(axis, exception):
    with exception:
        ...
```

Reply:
can do